quinn-interop/src/client.rs

```diff
@@ -62,7 +63,7 @@ fn test_case_implemented_or_exit() {
 }
 
 #[tokio::main(flavor = "multi_thread")]
-async fn main() -> Result<(), Box<dyn std::error::Error>> {
+async fn main() -> Result<()> {
```

Comment:
I've never loved shadowing `Result` this way, and usually like to keep it explicitly as `anyhow::Result` in return types. What do you think?

Reply:
I don't feel strongly; switched.